models/vgg/model.py

```diff
@@ -0,0 +1,54 @@
+"""Load VGG16 model and return it.
+
+Returns:
+    Module: Torch Module of VGG16.
+"""
+from torch.nn import Module
+from torchsummary import summary
+import torchvision.models as models
+
+
+class VGG16:
+    """Loads VGG16 model.
+    """
+    def __init__(self, batch_norm=False) -> None:
+        """Initialize model based on batch normalization parameter.
+
+        Args:
+            batch_norm (bool, optional): Include batch normalization layer. Defaults to False.
+        """
+        self.model = batch_norm
+        self._bn = batch_norm
+
+        # Set model to eval mode
+        self._model.cuda()
+        self._model.eval()
+
+    @property
+    def model(self) -> Module:
+        """Getter for the model
+
+        Returns:
+            Module: torch model
+        """
+        return self._model
+
+    @model.setter
+    def model(self, batch_norm: bool) -> None:
```

Comment:
Do we need this setter? Perhaps it'd be better to make this a private or protected method and call it in init?

Reply:
Sounds good. I will make the changes.